okgo/check/check_test.go

```diff
@@ -112,3 +116,18 @@ func TestRun_ErrorsButFilteredOut(t *testing.T) {
 	err := Run(projectParam, checkersToRun, []string{"p1"}, "dir", nil, 2, os.Stdout)
 	assert.NoError(t, err)
 }
+
+func TestRun_ErrorsOnTypeCheck(t *testing.T) {
+	projectParam := okgo.ProjectParam{
+		Checks: map[okgo.CheckerType]okgo.CheckerParam{
+			"error_on_type_check": {
+				Checker: &inMemoryChecker{checkerType: "error_on_type_check"},
+			},
+		},
+	}
+	checkersToRun := []okgo.CheckerType{
+		"error_on_type_check",
+	}
+	err := Run(projectParam, checkersToRun, nil, "dir", nil, 2, os.Stdout)
+	assert.Error(t, err)
```

Comment:
If the semantic goal is to check for the failure produced above, can we check that the error contains the string you return in the error at line 37? If the returned `err` doesn't have this, then could we pass in a `bytes.Buffer` writer or similar instead of `os.Stdout` at line 131 and verify that the command writes the expected error to the output stream?

Reply:
Fixed